addOns/authhelper/src/main/resources/db/migration/V1__Create_tables.sql

```diff
@@ -0,0 +1,65 @@
+CREATE CACHED TABLE IF NOT EXISTS "AUTHHELPER_DIAGNOSTIC" (
+    "CREATETIMESTAMP" TIMESTAMP NULL,
+    "ID" BIGINT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
+    "AUTHENTICATIONMETHOD" VARCHAR(4096) NOT NULL,
+    "CONTEXT" VARCHAR(4096) NOT NULL,
+    "USER" VARCHAR(4096) NOT NULL,
+);
+
+
+CREATE CACHED TABLE IF NOT EXISTS "AUTHHELPER_DIAGNOSTIC_STEP" (
+    "CREATETIMESTAMP" TIMESTAMP NULL,
+    "ID" BIGINT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
+    "DIAGNOSTICID" BIGINT NOT NULL,
+    "NUMBER" BIGINT NOT NULL,
+    "URL" VARCHAR(4096) NOT NULL,
+    "DESCRIPTION" VARCHAR(4096) NOT NULL,
+    "WEBELEMENTID" BIGINT NULL,
+);
+
+CREATE INDEX "AUTHHELPER_DIAGNOSTIC_STEP_DIAGNOSTICID_IDX" ON "AUTHHELPER_DIAGNOSTIC_STEP" ("DIAGNOSTICID");
+
+
+CREATE CACHED TABLE IF NOT EXISTS "AUTHHELPER_DIAGNOSTIC_MESSAGE" (
+    "CREATETIMESTAMP" TIMESTAMP NULL,
+    "ID" BIGINT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
+    "STEPID" BIGINT NOT NULL,
+    "NUMBER" BIGINT NOT NULL,
+    "MESSAGEID" BIGINT NOT NULL,
+);
+
+CREATE INDEX "AUTHHELPER_DIAGNOSTIC_MESSAGE_STEPID_IDX" ON "AUTHHELPER_DIAGNOSTIC_MESSAGE" ("STEPID");
+
+
+CREATE CACHED TABLE IF NOT EXISTS "AUTHHELPER_DIAGNOSTIC_SCREENSHOT" (
+    "CREATETIMESTAMP" TIMESTAMP NULL,
+    "ID" BIGINT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
+    "STEPID" BIGINT NOT NULL,
+    "DATA" VARCHAR(8388608) NOT NULL,
```

Comment:
Consider storing the image as a BLOB over VARCHAR with Base64 encoding? Could we have screenshots bigger than 8 MiB?

Reply:
As Base64 we can embed in the reports without further conversions. It's definitely possible but the default resolution is just under 1400x700 (which is less than 200KiB). But we'll get more data/numbers from auth tests.